vision/cloud-client/detect/vision_async_batch_annotate_images.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_async_batch_annotate_images]
+
+from google.cloud import vision_v1
+from google.cloud.vision_v1 import enums
+
+
+def sample_async_batch_annotate_images(
+    input_image_uri="gs://cloud-samples-data/vision/label/wakeupcat.jpg",
+    output_uri="gs://your-bucket/prefix/",
+):
+    """Perform async batch image annotation"""
+    # [START vision_async_batch_annotate_images_core]
+
+    client = vision_v1.ImageAnnotatorClient()
+
+    # input_image_uri = "gs://cloud-samples-data/vision/label/wakeupcat.jpg"
```

Comment:
(same in other samples)

Reply:
the function def line won’t show up in the docs.

Reply:
These were auto-generated, so there are 2 region tags, 
`vision_async_batch_annotate_images`
and 
`vision_async_batch_annotate_images_core` 
Depends on which one is used.

Reply:
That's fair enough, can we settle on one way to show them?

Reply:
The defaults were in the CLI functionality that I removed. As a comment, the user can't run as-is without editing. As function parameters, it won't show up in the docs. How do other samples handle this?

Reply:
There are two region tags here - `vision_async_batch_annotate_images` and `vision_async_batch_annotate_images_core `. If the first is used, then the default parameters are shown and the comment isn't needed. If the second is used, then the default parameters aren't needed but the comment is. 

Do we need both region tags? Why are we linking the sample differently from two different regions? I would like to avoid having the same values in two places if possible. 

Other samples usually use one pattern or the other. My preference is for using the outter region tags and removing the comments - showing imports is useful and it's preferable to have the default values being checked by the compiler.

Reply:
I verified that nothing live is pointing to the inner region tag before removing. That's a good point about compile-time verification.